plugins/esi/combo_handler.cc

```diff
@@ -187,6 +207,94 @@ InterceptData::~InterceptData()
   }
 }
 
+void
+CacheControlHeader::update(TSMBuffer bufp, TSMLoc hdr_loc)
+{
+  vector<string> values;
```

Comment:
Why a loop? If we use a loop then it's harder to set the `found_private` and `found_immutable` flags. I feel like unrolling the loop in this case is better since the # of parameters in Cache-Control is known.